### ISSUE TITLE
Crash with NullPointerException on Robot Box

### ISSUE THREAD

**Issue by SanAndreaP** [The-Fireplace/Mechanical-Soldiers#22]
Forge 14.23.0.2537
MechanicalSoldiers 1.1.4

I just crashed with a NullPointerException whilst just walking around:
```
java.lang.NullPointerException: Unexpected error
	at the_fireplace.mechsoldiers.tileentity.TileEntityRobotBox.getCPU(TileEntityRobotBox.java:94)
	at the_fireplace.mechsoldiers.blocks.BlockRobotBox.getPickBlock(BlockRobotBox.java:115)
	at mcjty.theoneprobe.rendering.OverlayRenderer.requestBlockInfo(OverlayRenderer.java:326)
	at mcjty.theoneprobe.rendering.OverlayRenderer.renderHUDBlock(OverlayRenderer.java:252)
	at mcjty.theoneprobe.rendering.OverlayRenderer.renderHUD(OverlayRenderer.java:117)
	at mcjty.theoneprobe.proxy.ClientProxy.renderGameOverlayEvent(ClientProxy.java:124)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_567_ClientProxy_renderGameOverlayEvent_Pre.invoke(.dynamic)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90)
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:179)
	at net.minecraftforge.client.GuiIngameForge.renderHUDText(GuiIngameForge.java:710)
	at net.minecraftforge.client.GuiIngameForge.func_175180_a(GuiIngameForge.java:172)
	at net.minecraft.client.renderer.EntityRenderer.func_181560_a(EntityRenderer.java:1098)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1117)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:397)
	at net.minecraft.client.main.Main.main(SourceFile:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
```

Here's the full log (warning: huge):
https://gist.githubusercontent.com/SanAndreasP/e94dea32e5ab51213699b981f9ddb416/raw/98020d7864869ae0ba3d64e805199b44d0dd9a99/fml-client-latest.log

**Comment by The-Fireplace** [comment]
I'm aware of the issue. For the moment, just don't look at the Robot Crate with TOP enabled. And perhaps break that crate, it is invalid, and won't ever spawn a robot. I'm working on the issue.

**Comment by SanAndreaP** [comment]
Thanks, I'll do that.

**Comment by The-Fireplace** [comment]
Fixed in 1.1.5
